doc/classes/InputMap.xml

```diff
@@ -78,6 +78,7 @@
 			<param index="0" name="action" type="StringName" />
 			<description>
 				Removes an action from the [InputMap].
+				[b]Warning:[/b] Removing built-in actions can lead to many control nodes not working properly. See also [method is_builtin_action].
```

Comment:
It may be worth to link the class directly here.
```suggestion
				[b]Warning:[/b] Removing built-in actions can lead to many [Control] nodes not working properly. See also [method is_builtin_action].
```